views/templates/admin/tabs/faviconConfiguration.tpl

```diff
@@ -0,0 +1,103 @@
+                        </div>
+                        <div>
+                            <div class="input-group fixed-width-lg">
+                                <span class="switch prestashop-switch fixed-width-lg">
+                                <input class="yes" type="radio" name="CHECKBOX_CUSTOMER" id="checkbox_track_new_customers_on" value="1" {if $bofavicon_params.CHECKBOX_CUSTOMER eq 1}checked="checked"{/if}>
+                                <label for="checkbox_track_new_customers_on" class="radioCheck">{l s='YES' d='Modules.Faviconnotificationbo.Admin'}</label>
+                                <input class="no" type="radio" name="CHECKBOX_CUSTOMER" id="checkbox_track_new_customers_off" value="0" {if $bofavicon_params.CHECKBOX_CUSTOMER eq 0}checked="checked"{/if}>
+                                <label for="checkbox_track_new_customers_off" class="radioCheck">{l s='NO' d='Modules.Faviconnotificationbo.Admin'}</label>
+                                <a class="slide-button btn"></a>
+                                </span>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <div class="control-label col-lg-5 col-md-4 col-xs-10">
+                            <label class="labelbutton">{l s='New Messages' d='Modules.Faviconnotificationbo.Admin'}</label>
+                        </div>
+                        <div>
+                            <div class="input-group fixed-width-lg">
+                                <span class="switch prestashop-switch fixed-width-lg">
+                                <input class="yes" type="radio" name="CHECKBOX_MESSAGE" id="checkbox_track_new_messages_on" value="1" {if $bofavicon_params.CHECKBOX_MESSAGE eq 1}checked="checked"{/if}>
+                                <label for="checkbox_track_new_messages_on" class="radioCheck">{l s='YES' d='Modules.Faviconnotificationbo.Admin'}</label>
+                                <input class="no" type="radio" name="CHECKBOX_MESSAGE" id="checkbox_track_new_messages_off" value="0" {if $bofavicon_params.CHECKBOX_MESSAGE eq 0}checked="checked"{/if}>
+                                <label for="checkbox_track_new_messages_off" class="radioCheck">{l s='NO' d='Modules.Faviconnotificationbo.Admin'}</label>
+                                <a class="slide-button btn"></a>
+                                </span>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </ol>
+        </div>
+        <div class="col-lg-5">
+            <div class="form-group">
+                <div id="divcolorpicker" class="control-label col-lg-6 col-md-4 col-xs-10">
+                    <label class="labelbutton" for="faviconbo_input_backgroundcolor">{l s='Notification Background Color' d='Modules.Faviconnotificationbo.Admin'}</label>
```

Comment:
Please, no useless upper cases. ;-)

Reply:
And it is mandatory to add the word _Notification_ here? If it is easily understandable for the merchants at this point of the user journey, perhaps we can delete the word.